### ISSUE TITLE
Looking-glass-client is preventing power management on host

### ISSUE THREAD

**Issue by lukecdavidson** [gnif/LookingGlass#158]
Host CPU: AMD FX-8350
Host GPU: AMD TAHITI Radeon R9 280
Guest GPU: AMD Ellesmere RX 580
Host Kernel version: 5.0.13
Host QEMU version: 4.0.0
powerdevil version: 5.15.5-1

Observed behavior: Looking-glass-client is preventing power management on host. Screen isn't switching off after the set timeout. Power Management widget displays "My SDL application is currently suppressing power management: Playing a game." Happens anytime the client is running.

Expected behavior: The screen to lock and turn off according to set power management.

**Comment by howaboutsynergy** [comment]
Seems like it should [already](https://github.com/gnif/LookingGlass/search?q=allowscreensaver&unscoped_q=allowscreensaver) allow screensaver by default, unless `-S` is specified? unless this `SDL_SetHint(SDL_HINT_VIDEO_ALLOW_SCREENSAVER, "1");` isn't enough, or others things are happening that are preventing ...

**Comment by lukecdavidson** [comment]
That is what I thought as well. I should have mentioned I am running the client without the -S option.

**Comment by gnif** [comment]
If this isn't working please consult the SDL project, we are already setting the correct and recommended hints to allow the screensaver/powersave to become active.

TL;DR; This is an issue with SDL2

**Comment by netboy3** [comment]
Fixed in [commit 1727c77](https://github.com/gnif/LookingGlass/commit/1727c7726bcc7e51dc2972f3ebc10f18aca8bc19)
